pycsob/client.py

```diff
@@ -17,6 +20,202 @@ def send(self, request, **kwargs):
         return super(HTTPAdapter, self).send(request, **kwargs)
 
 
+class ConvertMixin:
+    """Convert instance into ordered dict."""
+    
+    def _to_camel_case(self, name: str) -> str:
+        """Convert name to camel case form."""
+        parts = name.split("_")
+        return parts[0] + "".join(i.title() for i in parts[1:])
+
+    def _fields(self) -> Tuple[str, ...]:
```

Comment:
This function seems redundant to me. In my opinion, this is simpler:

```python
for field in fields(self):
    value = getattr(self, field.name)
```

Reply:
Removed as suggested (related also to [this).](https://github.com/zbohm/pycsob/pull/2#discussion_r1654638717)